src/main/java/net/jextra/fauxjo/Home.java

```diff
@@ -135,6 +135,7 @@ public boolean delete( T bean )
     }
 
     public String buildBasicSelect( String clause )
+        throws SQLException
```

Comment:
Also need to remove this `SQLException` throw clause, since it's no longer needed and will also cause breakage.

Reply:
It has already been removed in this commit: https://github.com/jextranet/fauxjo/pull/4/commits/d794db5ae041b88a41201301c5241f36f2517f06

Reply:
That commit only removed it from `buildBasicSelectStatement` it needs to be removed from `buildBasicSelect` as well.

Reply:
I see, it is removed now.